### PR TITLE
Badge colors issue, api titles issue

### PR DESCRIPTION
**Pull request by maxgrapps**
1) Api titles font-size decreased, font-weight changed to regular
was:
font_old
now:
font_now

2) Badge colors changed for each theme
blockscout, poa
blockscout, poa
eth
eth
etc
etc
rsk
rsk
xdai
xdai
sokol
sokol
ropsten
ropsten
goerli
goerli
rinkeby
rinkeby
kovan
kovan
lukso
lukso

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ## Current
 
 ### Features
-- [#2109](https://github.com/poanetwork/blockscout/pull/2109) - use bigger updates instead of `Multi` transactions in BlocksTransactionsMismatch
+- [#2138](https://github.com/poanetwork/blockscout/pull/2138) - use bigger updates instead of `Multi` transactions in BlocksTransactionsMismatch
```
Comment (vbaranov) [inline CHANGELOG.md]: @maxgrapps changes in CHANGELOG are wrong

**Comment by maxgrapps** [comment]
> @maxgrapps is this intentional, that different chains have the different number of variables set for api docs?

I changed some variables in _api.scss and _badge.scss for all themes. And add some variables for almost all themes files separately

File: CHANGELOG.md (hunk 1 above)
Comment (maxgrapps) [reply CHANGELOG.md]: What's wrong in changelog? In line 4? I don't added it. I just updated my fork to blockscout master

**Comment by vbaranov** [comment]
@maxgrapps 

>I changed some variables in _api.scss and _badge.scss for all themes. And add some variables for almost all themes files separately

Is this intentional that for Kovan there are 5 additions
```
$badge-success-color: #15bba6;
$badge-success-background-color: rgba(#15bba6, .1);
$badge-neutral-color: $tertiary;
$badge-neutral-background-color: rgba($tertiary, .1);
$api-text-monospace-color: $tertiary; 
```

and for example, for xDai, only 3
```
$badge-neutral-color: #20446e;
$badge-neutral-background-color: rgba(#20446e, .1);
$api-text-monospace-color: #20446e;
```

?

**Comment by maxgrapps** [comment]
> @maxgrapps
> 
> > I changed some variables in _api.scss and _badge.scss for all themes. And add some variables for almost all themes files separately
> 
> Is this intentional that for Kovan there are 5 additions
> 
> ```
> $badge-success-color: #15bba6;
> $badge-success-background-color: rgba(#15bba6, .1);
> $badge-neutral-color: $tertiary;
> $badge-neutral-background-color: rgba($tertiary, .1);
> $api-text-monospace-color: $tertiary; 
> ```
> 
> and for example, for xDai, only 3
> 
> ```
> $badge-neutral-color: #20446e;
> $badge-neutral-background-color: rgba(#20446e, .1);
> $api-text-monospace-color: #20446e;
> ```
> 
> ?

Yes, it's correct. Because it's impossible to add same logic for all themes. We reviewed each theme one by one and we decided, that we should change logic sometimes

File: CHANGELOG.md (hunk 1 above)
Comment (maxgrapps) [reply CHANGELOG.md]: I fixed changelog, added correct links to my pr and pr at the top, thanks

**Pull request merged**